CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpMenuViewController.swift

```diff
@@ -30,6 +30,7 @@ final public class HelpMenuViewController: UIViewController {
     private let giniConfiguration: GiniConfiguration
     private let tableRowHeight: CGFloat = 44
     private let margin: CGFloat = 16
```

Comment:
@chriskryn we need to rename the "general" margin and use more specific, what do you think about it?